blogpost2.css

```diff
@@ -0,0 +1,23 @@
+body{
+	background-color:white;
+}
+
+.title{
+	font-family:Courier;
```

Comment:
it's not safe to assume that every computer has Courier installed, so it's good to give a fallback font. Here's some info on that: https://stackoverflow.com/questions/4673963/css-fallback-fonts